.github/workflows/dest_bigquery.yml

```diff
@@ -0,0 +1,111 @@
+name: Destination Plugin BigQuery Workflow
+
+on:
+  pull_request:
+    paths:
+      - "plugins/destination/bigquery/**"
+      - ".github/workflows/dest_bigquery.yml"
+  push:
+    branches:
+      - main
+    paths:
+      - "plugins/destination/bigquery/**"
+      - ".github/workflows/dest_bigquery.yml"
+
+jobs:
+  resolve-runner:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    outputs:
+      runner: ${{ steps.resolve.outputs.runner }}
+    steps:
+      - name: Check if should use large runner
```

Comment:
let's remove this large runner as well. I don't think we need it works with the default runner fast enough given it's a small plugin otherwise it will block aws using big runner which I think it the only one that really needs that.